src/tensors/vector.jl

```diff
@@ -0,0 +1,45 @@
+containertype(::Type{T}) where T <:AbstractVector = T.name.wrapper
+
+unwrap(::Type{T}) where {T} = T
+
+
+single(::Type{Vector{S}}) where S = S[1]
+empty(::Type{Vector{S}}) where S = S[]
+
+postype(::Type{Vector{S}}) where {S} = Int
```

Comment:
why does vector need a postype?

Reply:
I think it doesn't but I was mirroring indextype for reasons unknown. Will check.

Reply:
Ah so this actually is how we determined the position type of element level. If you supply a vector that uses some other position type, then element level would have a different one and this would propagate up.

Reply:
Ah, maybe instead we can say `postype(::ElementLevel) = eltype(memtype(ElementLevel))`?